keps/sig-cloud-provider/20190422-cloud-controller-manager-migration.md

```diff
@@ -157,53 +159,35 @@ will prevent any of the v1.18 CCMs from claiming the lock. When the current hold
 
 ### Implementation Details/Notes/Constraints [optional]
 
-TODO after KEP summary and proposal is approved
+The cloud-provider needs to would need to implement an interface, that for a given controller returns the version of migration
```

Comment:
Can we add more details here please? Some questions off the top of my head:
* what does the interface look like?
* what components call it? 
* where is it implemented?